c-pallets/segment-book/src/lib.rs

```diff
@@ -622,6 +633,8 @@ pub mod pallet {
 		}
 
 		//Storage proof method
+		// Maybe expect a BoundedVec instead?
+		// You are trusting that the whatever is using the method has checked Vec limits already
 		fn storage_prove(acc: AccountOf<T>, prove_list: Vec<ProveInfo<T>>) -> DispatchResult {
```

Comment:
To be on the safe side, I chose `BoundedVec`:
https://github.com/CESSProject/cess/pull/88/files#diff-e315e38811b02c01aead9143bcfe8a7d0db178f24a154bad1431a08648393296R652